docs/sql-migration-guide-upgrade.md

```diff
@@ -122,7 +122,7 @@ license: |
 
   - In Spark version 2.4 and earlier, the `current_date` function returns the current date shifted according to the SQL config `spark.sql.session.timeZone`. Since Spark 3.0, the function always returns the current date in the `UTC` time zone.
```

Comment:
If date type is timezone independent, does `UTC time zone`  still make sense here?

Reply:
Yes, because getting from the time 'now' to a day still depends on a timezone. Right now it's April 9 2019 where I am but April 10 2019 elsewhere, even if those two dates themselves do not name a point in time without a timezone. So here we define the result to always be the day it is 'now' in UTC